content/en/docs/getting-started/hardware-requirements.md

```diff
@@ -5,49 +5,47 @@ description: "Define the hardware requirements for your Cozystack use case."
 weight: 5
 ---
 
-The Cozystack platform runs on top of Talos Linux, a minimalistic operating system designed solely to run Kubernetes.
-It is **not** possible to "share" a bare server with other services.
-
-The good news is that whatever other services you need to run, Cozystack can be the perfect platform to host them—securely,
-efficiently, and in a fully containerized or virtualized environment.
+Cozystack utilizes Talos Linux, a minimalistic Linux distribution designed solely to run Kubernetes.
+Usually it means that it will **not** be possible to share a server with services other than Cozystack runs.
+The good news is that whichever service you need, Cozystack will run it prefectly: securely, efficiently, and
+in a fully containerized or virtualized environment.
 
 Hardware requirements depend on your usage scenario.
-Below are several common deployment options—review them to determine what setup best fits your needs.
+Below are several common deployment options, review them to determine which setup fits best to your needs.
 
-## Proof-of-Concept Lab
+## Small lab
 
-Here are the baseline requirements for running a proof-of-concept installation.
+Here are the baseline requirements for running a tiny installation.
 
 **Compute:**
 
-- Three physical or virtual servers with amd64/x86_64 architecture, each having at least 16 GB RAM and 4 CPU cores.
-- If the servers are virtualized, ensure that nested virtualization is enabled and the CPU model is set to `host` (without emulation).
-- A PXE installation requires an extra management VM or physical server connected to the same network,
-  with any Linux system installed on it (for example, Ubuntu should be enough).
-  This VM should support the `x86-64-v2` architecture, which most probably can be achieved by setting CPU model to `host`.
+- Three physical or virtual servers with amd64/x86_64 architecture, with at least 16 GB RAM and 4 CPU cores each.
+- Virtualized servers need nested virtualization enabled and the CPU model set to `host` (without emulation).
+- PXE installation requires an extra management instance connected to the same network, with any Linux system able to run a Docker container.
```

Comment:
```suggestion
- PXE installation requires an extra management VM or physical server connected to the same network,
  with any Linux system able to run a Docker container.
```

Reply:
We refer to VM in the following list item. Let's keep it explicit that VM and physical server are valid options.

Reply:
After re-reading it for the fifth time, I finally understand what you mean. It shouldn't be a fresh VM or physical server. Most likely, it will be the regular workstation of an engineer who's doing the deployment.